src/main/resources/org/sagebionetworks/web/client/view/ProfileViewImpl.ui.xml

```diff
@@ -20,20 +20,20 @@
 							addStyleNames="fullWidthAlert">
 							<bh:Strong text="Please login to access this resource." />
 						</b:Alert>
-						<div class="margin-top-40">
```

Comment:
The new links panel needs to extend to the very top and bottom of the tab content area.